### PR TITLE
Ta 42255 42249

### PR DESCRIPTION
**Pull request by ApurwaMohite**
# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/dtm/test_rgw_s3_process_restart.py [hunk 1]
```diff
@@ -496,3 +497,149 @@ def test_continuous_write_during_rgw_s3_restart(self):
...
+        self.log.info("ENDED: Verify continuous DELETEs during rgw_s3 service restart using "
+                      "pkill")
+
+    # pylint: disable=too-many-statements
+    # pylint: disable-msg=too-many-locals
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-42255")
+    def test_ios_during_rc_rgw_restart(self):
+        """Verify IOs during RC pod rgw_s3 restart using pkill."""
```
Comment (priyankaborawake) [inline tests/dtm/test_rgw_s3_process_restart.py:589]: You can update the title if we are trying restart multiple times.

**Review by priyankaborawake — APPROVED** [review]
Minor comment added

**Review by sonalk0209 — APPROVED** [review]

**Comment by ApurwaMohite** [comment]
Collection logs:
[test_42255_42249_collect.log](https://github.com/Seagate/cortx-test/files/8896749/test_42255_42249_collect.log)

File: tests/dtm/test_rgw_s3_process_restart.py [hunk 2]
```diff
@@ -496,3 +497,152 @@ def test_continuous_write_during_rgw_s3_restart(self):
...
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-42249")
+    def test_cont_delete_during_rgw_svc_restart(self):
+        """Verify continuous DELETEs during rgw_s3 service restart using pkill."""
+        self.log.info("STARTED: Verify continuous DELETEs during rgw_s3 service restart using "
+                      "pkill")
+        test_prefix = 'test-42249'
+        wr_output = Queue()
+        test_cfg = DTM_CFG["test_42246"]
```
Comment (ravindrachoudhari) [inline tests/dtm/test_rgw_s3_process_restart.py:512]: Is it correct? do you want to take the config from test_42246?

Comment (ApurwaMohite) [reply tests/dtm/test_rgw_s3_process_restart.py:512]: Yes, this is correct.

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**